### ISSUE TITLE
[A11y]: SC Step 3 -New screen / updated flow - Request

### ISSUE THREAD

**Issue by kbermangestring** [department-of-veterans-affairs/va.gov-team#130201]
# 🔧 Accessibility Support Request

## I am requesting...

**Design / Research**

- [ x] Design review
- [ ] A11y annotations
- [ ] Prototype testing
- [ ] User research planning
- [ ] Tech support for research sessions

**Dev / QA / Training**

- [ ] A11y auditing & testing
- [ ] Collaboration Cycle accessibility testing artifact
- [ ] PR Review - *No need to fill this ticket out, just tag me as a reviewer*
- [ ] A11y training
- [ ] Office Hours / Deep Dive

## Background
- **Team name:** Decision Review
- **Product name:** Supplemental Claim
- **Feature name:** Step 3
- **Contact person:** @kbermangestring / @cinlackey 
- **Next Collaboration Cycle touchpoint & date if applicable:**
- **Link to PR/Figma/prototype etc:** [New screen added](https://www.figma.com/design/2LGebZcUuu5Iqh4QLPII6A/Supplemental-Claims--VA-0995-?node-id=7558-64279&t=0PgyHS8VmujWnLRy-11) &[ update to the user flow](https://www.figma.com/design/2LGebZcUuu5Iqh4QLPII6A/Supplemental-Claims--VA-0995-?node-id=7574-70038&t=0PgyHS8VmujWnLRy-11)
- **Links to any related GitHub issues:**
- **Additional details on your request:**

## Yes, I have

- [X ] Formatted the GitHub issue title to include: [A11y]: Product - Feature - Request
- [ ] Set my project under "Projects"
- [ X] Added the appropriate team/project label under "Labels" (e.g., Decision Reviews, BMT1, or BMT2)
- [ X] Verified that @bacitracin is assigned to this ticket
